.travis.yml

```diff
@@ -51,4 +51,5 @@ after_success:
 branches:
   only:
   - master
+  - istio-0.8
```

Comment:
delete this line - we don't want this branch name in travis anymore

Reply:
Done